docs/docs-beta/docs/guides/data-modeling/partitioning.md

```diff
@@ -1,6 +1,120 @@
 ---
-title: How to Partition Your Data
+title: Partitioning assets
 description: Learn how to partition your data in Dagster.
-sidebar_label: Partitioning pipelines
+sidebar_label: Partitioning assets
 sidebar_position: 30
 ---
+
+In Dagster, partitioning is a powerful technique for managing large datasets, improving pipeline performance, and enabling incremental processing. This guide will help you understand how to implement data partitioning in your Dagster projects.
+
+By the end of this guide, you'll have a comprehensive understanding of:
+
+- Defining partitions for Dagster assets and jobs
+- Establishing dependencies between partitioned assets
+- Leveraging Dagster partitions with external systems like dbt
+
+<details>
+  <summary>Prerequisites</summary>
+
+To follow the steps in this guide, you'll need:
+
+- A basic understanding of Dagster and assets. See the [Quick Start](/tutorial/quick-start) tutorial for an overview.
+- Basic understanding of data processing pipelines
+- Python 3.7 or later installed
+- Dagster library installed (`pip install dagster`)
+
+</details>
+
+## Define partitioned assets
+
+There are several ways to partition your data in Dagster:
+
+- [Time-based partitioning](#define-time-partitioned-assets), for processing data in specific time intervals
+- [Static partitioning](#define-partitions-with-predefined-categories), for dividing data based on predefined categories
+- [Two-dimensional partitioning](#define-two-dimensional-partitions), for partitioning data along two different axes simultaneously
+- [Dynamic partitioning](#define-partitions-with-dynamic-categories), for creating partitions based on runtime information
+
+### Define time-partitioned assets
+
+A common use case for partitioning is to process data that divides into time intervals, such as daily logs or monthly reports. Here's how to implement time-based partitioning in Dagster:
+
+<CodeExample filePath="guides/data-modeling/partitioning/time_based_partitioning.py" language="python" title="Time-based partitioning" />
+
+In this example:
+
+- We defined `daily_partitions` using `DailyPartitionsDefinition` with a start date of "2024-01-01". This will create a range of partitions from "2024-01-01" to the day before the current time.
+- The `daily_sales_data` asset is defined with this partitioning scheme.
+- The `daily_sales_summary` asset depends on `daily_sales_data` and also uses the same partitioning scheme.
+- The schedule `daily_sales_schedule` runs the job daily at 1:00 AM which partitions the data for the previous day.
```

Comment:
I think it is really helpful to have the schedule in this guide, since that is almost always the use case, and it really annoyed me that they were split across different concept pages before

Reply:
I also at first was surprised you used the manual way of launching a run request with a context key instead of `build_schedule_from_partitioned_job`, but honestly I kind of like the less magical approach after re-reading it a few times

Reply:
my thinking was later on, we could build on top of the manual `@schedule` easily to add examples like: kick off runs only during business dates/hours